### PR TITLE
Add svgo

### PR DESCRIPTION
**Pull request by hellatan**
the first commit is where the actually changes reside. the second commit is just formatting svgs with the updated space indentations.

### CODE REVIEW

File: .svgo-config.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "plugins":[
+    {"cleanupAttrs": true},
```
Comment (hellatan) [inline .svgo-config.json]: i only wanted this feature so i only enabled this plugin. These are all the current plugins and we can enable them as we see fit.

File: package.json [hunk 2]
```diff
@@ -9,15 +9,19 @@
...
     "postversion": "git push upstream master && git push --tags"
   },
-  "pre-commit": {
-    "run": [
-      "gitlint"
+  "lint-staged": {
+    "*.svg": [
+      "yarn format:svg"
+    ],
+    "*.js": [
```
Comment (hellatan) [inline package.json]: this was the `gitlint` script but i removed `pre-commit` in favor of `husky` and `lint-staged`

File: src/minus.svg [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <svg version="1.1" class="dibsvg-svg dibsvg-minus" viewBox="0 -120 250 250" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <rect width="250" height="15" rx="10"></rect>
-</svg>
\ No newline at end of file
+    <rect width="250" height="15" rx="10"/>
```
Comment (hellatan) [inline src/minus.svg]: 🤔 it does seem to self-close redundant tags though

**Comment by hellatan** [comment]
@nameofname @moriko thoughts?

**Comment by moriko** [comment]
I'm more than happy that the attributes would be cleaned up.
Have you tried out the `addAttributesToSVGElement`? As we have few attributes that should be added on svg this would be great if was enabled at some point.

**Comment by hellatan** [comment]
> Have you tried out the addAttributesToSVGElement?

@moriko i did not. I'll look into that today and report back

File: src/app-store.svg [hunk 4]
```diff
@@ -1,35 +1,33 @@
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="dibsvg-svg dibsvg-app-store" version="1.1" viewBox="0 -47 135 135">
-    <g>
```
Comment (hellatan) [inline src/app-store.svg]: these SVGs look the same even with the removed group tag

File: .svgo-config.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+quiet: true
```
Comment (hellatan) [inline .svgo-config.yml]: it seems like the preferred config is a yaml file

Comment (hellatan) [reply .svgo-config.yml]: @moriko i also did not look at all the plugins here, so please suggest any other ones we may want to use. These are the only ones that are currently on:
- cleanupAttrs
- addAttributesToSVGElement
- sortAttrs

File: src/app-store.svg (hunk 4 above)
Comment (moriko) [reply src/app-store.svg]: Grouping just add elements in folder in illustrator. If there's no additional attribute then it's save to remove.

**Comment by hellatan** [comment]
@nameofname @moriko ok, this is ready to be merged. I'm waiting for svgo to be published so that the `xmlns` attribute isn't always the first one on the `svg` node, but that can be updated once it has been published (https://github.com/svg/svgo/pull/984)

File: package.json [hunk 6]
```diff
@@ -50,22 +55,31 @@
     "dibs-scss-mixins": "^2.4.3",
     "dibs-scss-vars": "^3.6.4",
     "dibs-styles": "^2.4.0",
-    "dibslint": "^2.0.3",
```
Comment (nameofname) [inline package.json]: 👍

Comment (hellatan) [reply package.json]: linting #rocksolid

**Comment by hellatan** [comment]
`dibs-vg@1.73.0`

File: src/logo.svg [hunk 7]
```diff
@@ -1,10 +1,10 @@
-<svg version="1.1" class="dibsvg-svg dibsvg-logo" title="1stdibs.com" viewBox="543 138 2021.4 500" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (gedas-monginas) [inline src/logo.svg]: @hellatan do you think loosing a `title` from 1stdibs logo might be an issue? SEO? :)

Comment (hellatan) [reply src/logo.svg]: hmm...not sure tbh. i'll see to try and add it back.

Comment (hellatan) [reply src/logo.svg]: actually, it seem like there is no `title` attribute or svgs. a `<title>` element would have to be added to this svg instead so it seems like this attribute has been doing nothing for us. I'll add the `<title>` element though

Comment (hellatan) [reply src/logo.svg]: @gedas-monginas - https://github.com/1stdibs/dibs-vg/pull/158

**Pull request merged**